### ISSUE TITLE
hub create repository in organization without typing out the entire path

### ISSUE THREAD

**Issue by AlJohri** [github/hub#2443]
**The problem I'm trying to solve:**

I'm constantly creating repos in my organization and really hate having to type out the entire repo name, especially when they get fairly verbose. I would really like to do something like:

```
$ pwd
/Users/me/this-really-really-long-repo-name-that-I-dont-want-to-type
$ git create -p --org WPMedia
```

**How I imagine hub could expose this functionality:**

Unfortunately `-o` is taken up for "browse" so we'll have to go with `--org` instead. It would be great if you could just change browse to "-b" though although I understand that breaks backward compat.

**Comment by mislav** [comment]
Good idea! I welcome a PR that adds `--org` flag support as an alternative to the `hub create ORG/myrepo` syntax.
